build.sbt

```diff
@@ -61,7 +61,6 @@ lazy val cassandraBundle = project
     libraryDependencies += ("org.apache.cassandra" % "cassandra-all" % "3.11.3")
       .exclude("commons-logging", "commons-logging"),
     dependencyOverrides += "com.github.jbellis" % "jamm" % "0.3.3", // See jamm comment in https://issues.apache.org/jira/browse/CASSANDRA-9608
-    assembly / target := target.value / "bundle" / "pekko" / "persistence" / "cassandra" / "launcher",
```

Comment:
thanks for looking into this! while you could say moving the jar around would be an incompatible change, on the other hand you could say the bundle is a testkit component anyway, and we don't promise compatibility for testkit components (https://pekko.apache.org/docs/pekko/current/common/binary-compatibility-rules.html), so this might be fine.

Reply:
We could keep the same path, but `sbt-assembly` seems like has a bug. https://github.com/sbt/sbt-assembly/issues/503

Reply:
oh interesting! as said, changing the path might be fine given it's a test component, though.

Reply:
OK, let we moving on.